src/features/user/MFV2/InfoAndCTA/SDGCardList/SingleSDGCard.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactElement } from 'react';
+import styles from '../InfoAndCta.module.scss';
+import { useTranslations } from 'next-intl';
+
+interface Props {
+  title: string;
+  icon: ReactElement;
+  index: number;
+  color: string;
+}
+
+const SingleSDGCard = ({ title, icon, index, color }: Props) => {
+  const t = useTranslations('Profile');
+  return (
+    <div
+      className={styles.singleSDGCardContainer}
+      style={{ background: color }}
+    >
+      <div>
+        <span className={styles.singleCardIndex}>
+          {String(index + 1).padStart(2, '0')}
```

Comment:
For readability, could you create a helper function for this?

Reply:
Not needed anymore